templates/include/glean/glean.2.parquetmr.txt

```diff
@@ -267,7 +267,7 @@ message baseline {
   }
   optional group events (LIST) {
     repeated group list {
-      required group element @COMMON_EVENT_1_TXT@
+      required group element @GLEAN_EVENT_1_TXT@
```

Comment:
Since this is now only user in 1 place, we could remove `include/glean/event.1.parquetmr.txt` and insert the schema directly here.